printer_mods/42bios/relay_mount/README.md

```diff
@@ -0,0 +1,9 @@
+Simple relay-mount for 2x [2 channel relay modules](https://www.makershop.de/en/module/relais/2-kanal-relais/) - e.g. for [moonraker power plugin.](https://github.com/Arksine/moonraker/blob/master/docs/web_api.md#power-apis)<br>
+Mount with two M5x8 screws and T- or hammerhead nuts to normal 2020 extrusions.<br>
+4x or 8x M2x8 self tapping screws to mount the relay module.
+<br>
+
+<br>
+
+version: 0.1<br>
+![GitHub Logo](https://i.imgur.com/qMGkSe8.png)![GitHub Logo](https://i.imgur.com/2PARhpQ.png)![GitHub Logo](https://i.imgur.com/p3H5AJV.jpg)
```

Comment:
Can you add the images to your PR instead of referencing them on imgur?  That way the mod is self-contained here and doesn't depend on external services.  Also change the alt text from `GitHub Logo` to something that describes each image.

Reply:
I hope its okay now?